.mergify.yml

```diff
@@ -21,27 +21,27 @@ pull_request_rules:
       - author=mergify[bot]
       - base=release-1.6
       - label!=do-not-merge
-      - 'status-success=DCO'
-      - 'check-success=canary'
-      - 'check-success=unittests'
-      - 'check-success=golangci-lint'
-      - 'check-success=codegen'
-      - 'check-success=lint'
-      - 'check-success=modcheck'
-      - 'check-success=pvc'
-      - 'check-success=pvc-db'
-      - 'check-success=pvc-db-wal'
-      - 'check-success=encryption-pvc'
-      - 'check-success=encryption-pvc-db'
-      - 'check-success=encryption-pvc-db-wal'
-      - 'check-success=encryption-pvc-kms-vault-token-auth'
-      - 'check-success=TestCephSmokeSuite (v1.16.15)'
-      - 'check-success=TestCephSmokeSuite (v1.21.0)'
-      - 'check-success=TestCephHelmSuite (v1.16.15)'
-      - 'check-success=TestCephHelmSuite (v1.22.0)'
-      - 'check-success=TestCephMultiClusterDeploySuite (v1.21.0)'
-      - 'check-success=TestCephUpgradeSuite (v1.16.15)'
-      - 'check-success=TestCephUpgradeSuite (v1.21.0)'
+      - "status-success=DCO"
```

Comment:
Were these quotes intentionally changed, or just from the editor? Not sure it matters, but hopefully they don't change again from someone else's editor. :)